tests/test-helper.js

```diff
@@ -1,7 +1,9 @@
-import Application from '../app';
-import config from '../config/environment';
-import { setApplication } from '@ember/test-helpers';
-import { start } from 'ember-qunit';
+import Application from "../app";
+import config from "../config/environment";
+import { setApplication } from "@ember/test-helpers";
+import { start } from "ember-qunit";
+
+import "ember-cli-testdouble-qunit";
```

Comment:
I created this addon and the library it wraps, `testdouble-qunit`, which just adds the `assert.verify` method used below, which mirror `td.verify` but reports more nicely into the QUnit test results and doesn't blow up the whole test on a single failure. Also removes the need to use `assert.expect(0)`.